### ISSUE TITLE
Problems in simulation data

### ISSUE THREAD

**Issue by Cicilalal** [theislab/scarches#106]
Hi, I don't know why I don't do well in simulation data using the base model of scVI and scANVI.  Batch8 and Batch9 are target data, which cannot be well integrated with source data  after fine-tuning.  I have adjusted the learning rate, but it didn't work. Are there any other parameters that need to be adjusted?  I have no idea, can you give me some advice? Thank you in advance！
![24C KJTR6Z(39LXW0XNLW(W](https://user-images.githubusercontent.com/99342457/163101205-aea67bde-5cfa-4137-9600-ec68c2b39d2c.png)

**Comment by M0hammadL** [comment]
hi. please start an issue in the reproducibility not here. 

thx
